rstar/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Added
 - `RTree::nearest_neighbors` method based on
   [spade crate's implementation](https://github.com/Stoeoef/spade)
+- Added a `SelectionWithDataIterator` trait.
```

Comment:
I think the trait is `SelectionFunctionWithData`, isn't it? I would suggest referencing the primary API entry point `` here as well to help people who read the change log and want to try this out.